interface/LeptonEfficiencySF.h

```diff
@@ -863,22 +887,28 @@ class LeptonEfficiencySF
         }else if(id==169){
             if (is2016){
                 if(etaCen<0.9){
-                    if(etaFwd<0.9){ eff.first=0.9490; eff.second=0.001;
-                    }else if(etaFwd<1.2){ eff.first=0.9628; eff.second=0.005;
-                    }else if(etaFwd<2.1){ eff.first=0.9477; eff.second=0.005;
-                    }else if(etaFwd<2.4){ eff.first=0.9420; eff.second=0.010;
+                    if(etaFwd<0.9){ eff.first=0.9956; eff.second=0.0865;
```

Comment:
Hi Hugues,

Okay I will update them!
But I want to ask one thing. We never used eff.second (error) in our code.
Do anybody know where this has been used.

for the time being i guess it would not effect but yes I will update them.